### PR TITLE
BUG FIX: Fatal Error when cancelling subscription

### PR DESCRIPTION
**Pull request by eighty20results**

### CODE REVIEW

File: pmpro-woocommerce.php [hunk 1]
```diff
@@ -204,6 +204,9 @@ function pmprowoo_activated_subscription($subscription)
     if(empty($pmprowoo_product_levels))
         return;
 	
+    if ( is_numeric( $subscription ) {
```
Comment (jorisberg) [inline pmpro-woocommerce.php]: parse error, missing closing bracket line 207

**Comment by jorisberg** [comment]
Checked at live site. Two problems:

(1) Parse error. Updated snippet:
`if ( is_numeric( $subscription )) {
     $subscription = wcs_get_subscription( $subscription );
    }`

(2) Snippet (also) needs to be inserted in the pmprowoo_cancelled_subscription function at line 250 (reviewed version) or line 247 (current version). 

With these updates pending cancellation subscription is processed correctly to cancelled at expiry date.

**Comment by eighty20results** [comment]
Should fix Issue #62

**Comment by contemplate** [comment]
Why has this not been incorporated into the plugin yet @strangerstudios ?  We just realized all my  client's member subscriptions aren't cancelling after the "pending cancel" stage.  Glad I found this patch.  Any ETA on what this will be added?

**Pull request merged**